iai_apartment/launch/apartment_display.launch

```diff
@@ -3,7 +3,7 @@
   <include file="$(find iai_apartment)/launch/apartment_bringup.launch" />
   <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui">
     <remap from="robot_description" to="apartment_description" />
-    <remap from="joint_states" to="apartment_joint_states" />
+    <remap from="joint_states" to="iai_apartment/cram_joint_states" />
```

Comment:
Can you call it iai_kitchen? For consistency. Maybe we should rename to something proper but I can't be bothered now :)

Reply:
I don't care what the topic is called. Thought the convention was to name it after the package.

Reply:
No, it's a hard-coded string. So please change if you don't care.